controller/array_action/array_mediator_ds8k.py

```diff
@@ -11,6 +11,9 @@
 from controller.array_action import config
 from controller.array_action.array_action_types import Volume
 
+LOGIN_PORT_WWPN = 'wwpn'
+LOGIN_PORT_STATE = 'state'
+LOGIN_PORT_STATE_ONLINE = 'online'
```

Comment:
remove the replaced const `IOPORT_STATUS_ONLINE`

Reply:
`IOPORT_STATUS_ONLINE` does not equal to `'state'`, it's equal to `'online'`, please remove it, even though it is not yours. thanks.

Reply:
Please comment on the correct line next time.

Reply:
I cannot commit outside the diff

Reply:
this i why i didn't push resolve before pushing